src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  box-sizing: border-box;
```

Comment:
it is common practice to apply `box-sizing: border-box` for all elements via `*` selector

Reply:
resolved